src/datanode/src/error.rs

```diff
@@ -402,6 +402,9 @@ pub enum Error {
         #[snafu(implicit)]
         location: Location,
     },
+
+    #[snafu(display("Not yet implemented: {what}"))]
+    NotYetImplemented { what: String },
```

Comment:
Maybe `Unsupported`? This is shorter.

Reply:
Was `Unsupported` but on second thought `NotYetImplemented` is better, because it's softer, representing something that will be implemented in the future. While `Unsupported` is, well, unsupported, and just can't directly know whether it will be supported in the future.